### ISSUE TITLE
Add Vet Center schema in content model documentation

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9764]

**Comment by kevwalsh** [comment]
@msbtterswrth this is awesome... i wonder how we can track implemented schema.org vs _designed_. What if the checkbox was for "this is done" vs the Schema property was "this is what it should be"? Then the view you linked to can be filtered by "Schema property" IS NOT EMPTY?

**Comment by msbtterswrth** [comment]
Yep that sounds good. I just made that change so you can refresh and should see it updated now.

**Comment by kevwalsh** [comment]
Reposting [the documentation](https://airtable.com/shrbYcgAlGuKL0WdU) and closing this as done!
